src/main/java/Mimi.java

```diff
@@ -0,0 +1,300 @@
+import java.util.ArrayList;
+import java.io.*;
+import java.util.Scanner;
+import java.util.Collection;
+
+public class Mimi {
+    static final String bar = "--------------------------------------------------";
+    static ArrayList<Task> tasks = new ArrayList<>();
+    static String fileName = "data/out.txt";
+
+    public static void main(String[] args) {
+
+        greet();
+        Scanner sc = new Scanner(System.in);
+        String userinput;
+        try {
+            loadTasks(new File(fileName));
+        } catch (MimiException e) {
+            System.out.println(e.getMessage());
+        }
+
+        while (true) {
+            System.out.print("> ");
+            userinput = sc.nextLine();
+            System.out.println(bar);
+
+            // exit if the user types something like 'quit' or 'exit
```

Comment:
invalid comment. you are only checking for `exit` not `quit`. so the comment is tripping the reader!